From db531373e81c68198ca6170f1e54ea966c0ce2ab Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Fri, 29 Oct 2021 18:50:04 -0500
Subject: [PATCH 2/2] Fix more ntpq man page formatting

The previous fix only fixed this in asciidoc, not asciidoctor.
Also, there were other examples of the same type of text that either
needed fixing or were not formatted consistently.

Signed-off-by: Richard Laager <rlaager@wiktel.com>
---
 docs/includes/ntpq-body.adoc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/docs/includes/ntpq-body.adoc b/docs/includes/ntpq-body.adoc
index 90e2f19b3..92c2359ae 100644
--- a/docs/includes/ntpq-body.adoc
+++ b/docs/includes/ntpq-body.adoc
@@ -307,7 +307,7 @@ ind assid status conf reach auth condition last_event cnt
   server so loaded that none of its MRU entries age out before they
   are shipped. With this option, each segment is reported as it arrives.
 
-[[mrulist]]+mrulist+ [+limited+ | +kod+ | +mincount=+'count' | +mindrop=+'drop' | +minscore=+'score' | +maxlstint=+'seconds' | +minlstint=+'seconds' | +laddr=+'localaddr' | +sort=+'sortorder' | +resany=+'hexmask' | +resall=+'hexmask' | +limit=+'limit' | +addr.'num'=+'address']::
+[[mrulist]]+mrulist+ [+limited+ | +kod+ | +mincount=+'count' | +mindrop=+'drop' | +minscore=+'score' | +maxlstint=+'seconds' | +minlstint=+'seconds' | +laddr=+'localaddr' | +sort=+'sortorder' | +resany=+'hexmask' | +resall=+'hexmask' | +limit=+'limit' | +addr.+'num'+=+'address']::
   Obtain and print traffic counts collected and maintained by the
   monitor facility. This is useful for tracking who _uses_ or
   _abuses_ your server.
@@ -316,19 +316,19 @@ Except for +sort=+'sortorder', the options
 filter the list returned by +ntpd+. The +limited+ and +kod+ options
 return only entries representing client addresses from which the last
 packet received triggered either discarding or a KoD response.
-the +addr.'num'=+ option adds specific addresses to retrieve when
+the +addr.+'num'+=+ option adds specific addresses to retrieve when
 +limit=+'1'. Values of 0 to 15 are supported for 'num'. Also, used
-internally with +last.'num'=+'hextime' to select the starting point
+internally with +last.+'num'+=+'hextime' to select the starting point
 for retrieving continued response.
 the +frags=+'frags' option limits the number of datagrams
 (fragments) in response.  Used by newer ntpq versions instead
 of limit= when retrieving multiple entries.
 The +limit=+ option limits the MRU entries returned per response.
 limit=1 is a special case:  Instead of fetching beginning with
-the supplied starting points (provided by a last.x and addr.x
+the supplied starting points (provided by a last.'x' and addr.'x'
 where 0 <= x <= 15, default the beginning of time) newer neighbor,
 fetch the supplied entries. This enables fetching multiple entries
-from given IP addresses (provided by addr.x= entries where 0 <= x
+from given IP addresses (provided by addr.'x'= entries where 0 <= x
 <= 15). When limit is not one and frags= is provided,
 the fragment limit controls. NOTE: a single mrulist command may
 cause many query/response rounds allowing limits as low as 3 to
-- 
2.25.1

